pkg/result/ignore.go

```diff
@@ -184,6 +184,9 @@ func (c *IgnoreConfig) MatchLicense(licenseID, filePath string) *IgnoreFinding {
 func ParseIgnoreFile(ctx context.Context, ignoreFile string) (IgnoreConfig, error) {
 	var conf IgnoreConfig
 	if _, err := os.Stat(ignoreFile); errors.Is(err, fs.ErrNotExist) {
+		if ignoreFile != DefaultIgnoreFile {
+			return IgnoreConfig{}, xerrors.Errorf("%s does not exist", ignoreFile)
```

Comment:
Should we error out or just simply `WARN` the user that their ignorefile doesn't exist? @knqyf263 WDYT?

Reply:
If I may, while a warning might be enough when a user calls trivy explicitly, in an automated context, unless there's some warning detection put in place, it might get missed and thus the scan output will not match expectation.

Reply:
Fair enough. It looks better to return an error.